src/SFML/Window/Unix/JoystickImpl.cpp

```diff
@@ -398,8 +398,7 @@ std::string getJoystickName(unsigned int index)
     if (fd >= 0)
     {
         // Get the name
-        char name[128];
-        std::memset(name, 0, sizeof(name));
+        char name[128] = {0};
```

Comment:
This could be simply `char name[128] = {};` or `char name[128] = "";`.

Reply:
Fixed, thanks